src/index.ts

```diff
@@ -30,6 +30,9 @@ type IReceivedError = Partial<Error> & string & {
 };
 
 type ResponseFormat = 'json' | 'text' | 'html';
+interface ErrorReporterOptions {
+    addStackOverflowLink?: boolean;
```

Comment:
Remove the question mark. Then in the exported function `errorReporter`, use `Partial<ErrorReporterOptions>`.

Reply:
fixed